### PR TITLE
PICA-133: Privacy Policy and TOU pages

### PR DESCRIPTION
**Pull request by easteregg**
This PR introduces Privacy Policy and Terms of Use pages as per CU task. 
As well as adding links to these pages from wallet connection.

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [PICA-133 Add Terms of Services, Privacy Policy + Add 'By connecting wallet' line](https://app.clickup.com/t/20465559/PICA-133)

File: frontend/apps/picasso/pages/terms-of-use.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              <ListItem>Reserved Rights</ListItem>
+              <ReservedRights />
+              <ListItem>Assignment</ListItem>
+              <Assignment />
+              <ListItem>Governing Law and Venue</ListItem>
+              <GoverningLawAndVenue />
+              <ListItem>Entire Agreement</ListItem>
+              <EntireAgreement />
+              <ListItem>Access and Acceptance</ListItem>
+              <AccessAndAcceptance />
```
Comment (fl-y) [inline frontend/apps/picasso/pages/terms-of-use.tsx:94]: I don't really understand the reasoning behind splitting these into a separate file

Comment (easteregg) [reply frontend/apps/picasso/pages/terms-of-use.tsx:94]: Will bring them back into a single file. thanks!

**Comment by dzmitry-lahoda** [comment]
@fl-y

**Comment by dzmitry-lahoda** [comment]
@fl-y

**Pull request closed without merging**